### PR TITLE
Improving docs...

### PR DESCRIPTION
**Pull request by jlopezdb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,83 @@
...
+  ...
+  ...
+
+```
+
+Explain use:
+------------
+
+You can find more information into each roles, but we are going to do a little abstract for you:
+  * If you want a yarn cluster, you must configure in vars master:
```
Comment (roberveral-datio) [inline README.md]: YARN (caps again xD)

"you must set, in `master:vars`:"

File: README.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+     - hadoop_is_namenode: true
+     - hadoop_is_resourcemanager=true
+     - spark_is_master=false
+  and worker vars:
```
Comment (roberveral-datio) [inline README.md]: and in `worker:vars`

File: README.md [hunk 3]
```diff
@@ -1 +1,83 @@
...
+     - spark_is_worker=false
+  * If you want a spark standalone cluster, you must configure in vars master:
+     - hadoop_is_namenode: true
+     - hadoop_is_resourcemanager=false
+     - spark_is_master=true
+  and worker vars:
+     - hadoop_is_datanode=true
+     - hadoop_is_nodemanager=false
+     - spark_is_worker=true  
+
```
Comment (roberveral-datio) [inline README.md]: Maybe all this part can be clearer:

* Spark cluster using YARN cluster manager with HDFS

```
[masters:vars]
hadoop_is_namenode=true
hadoop_is_resourcemanager=true
spark_is_master=false

[workers:vars]
hadoop_is_datanode=true
hadoop_is_nodemanager=true
spark_is_worker=false
```

* Spark cluster using Standalone cluster manager with HDFS

```
[masters:vars]
hadoop_is_namenode=true
hadoop_is_resourcemanager=false
spark_is_master=true

[workers:vars]
hadoop_is_datanode=true
hadoop_is_nodemanager=false
spark_is_worker=true
```

File: ansible/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+Raspark
```
Comment (roberveral-datio) [inline ansible/README.md]: Same as before

**Comment by jlopezdb** [comment]
@roberveral-datio comments done

**Pull request merged**